### ISSUE TITLE
searchVertical does not respect offset option.

### ISSUE THREAD

**Issue by darionco** [goferito/node-bing-api#33]
Sorry for the quick report.

file: bing.js
line: 69
function: `searchVertical`

is:
```
+ (opts.offset ? "&offset=" + opts.skip : "")
```
should be either:
```
+ (opts.offset ? "&offset=" + opts.offset : "")
```
or:
```
+ (opts.skip ? "&offset=" + opts.skip : "")
```

**Comment by darionco** [comment]
oh I just saw #29

**Comment by goferito** [comment]
Yup, it's fixed on v3.2.1,  published an hour ago :smile: Thanks for the report anyway. Don't forget to :star:  the repo :smile:
